power/03_influx.yaml

```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: power
+  name: influxdb
+  labels:
+    component: influxdb
```

Comment:
I dont think component is a useful label in this context. labels are all a bit wishy-washy so do what you want with them. there are these long and awful standard label names - i ignore them. labels should be convenient.

Reply:
removed.